app/styles/_colors.scss

```diff
@@ -1,3 +1,7 @@
 .bc-t-bg-grey-4 {
   background-color: $color_nhsuk-grey-4;
 }
+
+.bc-t-border-top--blue {
+  border-top: 2px solid $color_nhsuk-blue;
+}
```

Comment:
Isn't this just for colours?

Reply:
up for debate but I thought it belonged here more than in _app

Reply:
Just wanted to check really. In WPF world I would have put this style in a file named borders. Keep it in here for now, can review later.

Reply:
yeah I thought that would be overkill for now!!